app/views/staff/reports/_summary_short.html.haml

```diff
@@ -0,0 +1,17 @@
+%dl.govuk-summary-list.govuk-summary-list
+  - if current_user.service_owner?
+    .govuk-summary-list__row
+      %dt.govuk-summary-list__key
+        = t("form.label.report.organisation")
+      %dd.govuk-summary-list__value
+        = report.organisation.name
+  .govuk-summary-list__row
+    %dt.govuk-summary-list__key
+      = t("form.label.report.deadline")
+    %dd.govuk-summary-list__value
+      = report.deadline
+  .govuk-summary-list__row
+    %dt.govuk-summary-list__key
+      = t("form.label.report.state")
+    %dd.govuk-summary-list__value
+      = report.state
```

Comment:
This is a useful summary to explain the context of the comment. Are you planning a feature spec to explain the benefit?